index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Global Summit</title>
+</head>
+<body>
+
+        <header>
+            <div class="upper-links">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <a href="#">Logout</a>
+                <a href="#">Mypage</a>
+                <a href="#">Korean</a>
+            </div>
+            <nav class="nav-bar">
+                    <div id="logo"><img src="./images/summit-icon.jpg"></div>
```

Comment:
- [ ] Kindly make sure the logo is a link that leadss to the home page as per the project requirements :recycle:

Reply:
I've made sure by changing the logo as a link that leads to home page now